packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -585,6 +234,44 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
       }
     );
   }
+
+  /**
+   *
+   */
+  private createLine() {
+    const lineId = uuid();
```

Comment:
this action we can perform inside LIne class constructor

Reply:
Done